composer.json

```diff
@@ -107,6 +107,7 @@
         "amphp/amp": "^2.5",
         "amphp/http-client": "^4.2.1",
         "amphp/http-tunnel": "^1.0",
+        "async-aws/core": "^1.5",
```

Comment:
We need a conflict rule forbidding `<1.5` then)

Reply:
Sure. 

Just on the root composer.json or the bridge's composer.json too?

Reply:
It's a `require` rule in the bridge one, all good :)